server/api/users/users.controller.ts

```diff
@@ -92,10 +92,32 @@ const updateUserById: RequestHandler<{ id: string }, string, UpdatableFields> =
   res.sendStatus(200);
 };
 
+interface FlagAttributes {
+  id: number;
+  user_id: number;
+  flag_name: string;
+}
+
+const getUserFlags: RequestHandler<{ id: string }, { message: string } | { flags: string[] }> = async (req, res) => {
```

Comment:
this gets _any_ user's flags, but i think we only ever want to expose a user's own flags - so this route should probably not have an id parameter, and only return the flags of the currently-logged-in-user.